src/screens/FinancialSummary/components/FinancialInfo.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Typography, Button, Divider } from '@material-ui/core';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import TextField from '../../../components/TextField';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      color: theme.palette.text.primary,
+    },
+    content: {
+      margin: 10,
+      border: `1px solid ${theme.palette.divider}`,
+      borderRadius: '6px',
+      padding: '20px',
+    },
+    itemWrapper: {
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+    },
+    right: {
+      textAlign: 'right',
+    },
+    bolded: {
+      fontWeight: 700,
+    },
+    divider: {
+      margin: '10px 0px',
+    },
+    balanceText: {
+      fontWeight: 700,
+      color: theme.palette.primary.main,
+    },
+    fieldContainer: {
+      marginTop: '10px',
+    },
+  }),
+);
+
+interface CardProps {
+  bankName: string;
+  accountNumber: number;
+  accountName: string;
+  balance: number;
+}
+
+export default function FinancialInfo(props: CardProps) {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <Typography variant="h2" color="textPrimary">
+        Financial Info
+      </Typography>
+      <div className={classes.content}>
+        <div className={classes.itemWrapper}>
+          <div>
+            <Typography className={classes.bolded} variant="body1">
+              Bank
+            </Typography>
+            <Typography className={classes.bolded} variant="body1">
+              Account Number
+            </Typography>
+            <Typography className={classes.bolded} variant="body1">
+              Account Name
+            </Typography>
+          </div>
+          <div className={classes.right}>
+            <Typography variant="body1">{props.bankName}</Typography>
+            <Typography variant="body1">**** **** {props.accountNumber}</Typography>
```

Comment:
Is this ****** temporary? I'm curious how sensitive this number is/how we should handle this.

Reply:
True, I think on the Adalo app it showed the entire accounting number. I don't think it's a big deal either way, though.